code/src/components/input.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import tasks from "reducers/todo";
+import styled from "styled-components";
+import uniqid from "uniqid";
+
+const SubmitButton = styled.button`
+  border-radius: 8px;
+  padding: 10px 15px;
+  font-family: "Architects Daughter", sans-serif;
+  font-size: 0.9rem;
+`;
+
+const InputArea = styled.input`
+  border: 0;
+  border-bottom: 3px dashed #fdcb6e;
+  background-color: transparent;
+  font-family: "Architects Daughter", sans-serif;
+  font-size: 1.3rem;
+  padding: 10px 15px;
+  width: 30%;
+  color: #494a4b;
+`;
+
+const AddTodo = () => {
+  const [inputValue, setInputValue] = useState("");
+
+  const dispatch = useDispatch();
+
+  const onFormSubmit = (event) => {
+    event.preventDefault();
+
+    const newTask = {
+      id: uniqid(),
+      text: inputValue,
+      complete: false,
+    };
+
+    dispatch(tasks.actions.addItem(newTask));
+    setInputValue("");
```

Comment:
Good solution with uniqid and to create a new task with the initial value of false